### PR TITLE
Linux arm64 Docker

### PR DESCRIPTION
**Pull request by jsirianni**
## Description of Changes

Add docker support for linux/ARM64
- added goos and goarch to version output
- use `buildx` plugin when building
- refactor docker commands into single docker command for build, tag, push

This will allow a single tag to be built for AMD64 and ARM64.

## **Please check that the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Add a changelog entry (for non-trivial bug fixes / features)
- [x] CI passes

### CODE REVIEW

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.4827633|-0.15521646|131.1343|+1.8952026|
|1|5000|5.258776|+4.2915344e-06|137.69383|-3.0373077|
|1|10000|11.1037|+0.81050587|144.71269|-1.108429|
|1|50000|54.257076|-2.3667908|181.11598|+4.404648|
|1|100000|96.91557|-4.0941544|226.21107|-13.528961|
|10|100|1.7931633|-0.46552324|132.6863|-0.7580719|
|10|500|6.3449435|+0.31032753|140.59941|+1.4210663|
|10|1000|12.034595|-0.017159462|147.05658|+2.2684631|
|10|5000|56.810886|+1.707077|180.27654|-1.4723816|
|10|10000|102.690895|-4.6140594|221.55132|-0.7095947|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.5172448|-0.12073493|127.68696|-1.5521317|
|1|5000|5.7415676|+0.48279572|137.25862|-3.472519|
|1|10000|10.448501|+0.15530682|142.59604|-3.2250824|
|1|50000|50.553738|-6.0701294|175.73532|-0.9760132|
|1|100000|91.71665|-9.293076|237.94316|-1.796875|
|10|100|1.8448703|-0.4138162|134.81317|+1.3688049|
|10|500|6.3277135|+0.2930975|141.4317|+2.253357|
|10|1000|11.82793|-0.22382355|148.63348|+3.8453674|
|10|5000|54.811935|-0.29187393|182.23936|+0.490448|
|10|10000|122.54909|+15.244133|232.0058|+9.744888|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.500019|-0.13796079|128.22455|-1.0145416|
|1|5000|5.2758727|+0.017100811|137.75957|-2.9715729|
|1|10000|10.413981|+0.12078762|144.30011|-1.5210114|
|1|50000|51.01617|-5.6076965|170.79176|-5.919571|
|1|100000|100.82727|-0.18245697|230.35399|-9.386047|
|10|100|2.258732|+4.553795e-05|133.78786|+0.3434906|
|10|500|6.068999|+0.03438282|139.82301|+0.6446686|
|10|1000|11.8278475|-0.22390652|149.04121|+4.2530975|
|10|5000|58.650543|+3.5467339|182.132|+0.38308716|
|10|10000|107.5161|+0.2111435|228.39804|+6.1371307|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.4999975|-0.13798225|127.68373|-1.5553589|
|1|5000|5.4485707|+0.18979883|139.44989|-1.28125|
|1|10000|10.086253|-0.20694065|144.88268|-0.93844604|
|1|50000|51.953053|-4.6708145|174.5202|-2.1911316|
|1|100000|101.28346|+0.27373505|238.16217|-1.5778656|
|10|100|1.9310526|-0.32763398|133.31331|-0.13105774|
|10|500|7.000095|+0.9654789|138.91487|-0.2634735|
|10|1000|11.706934|-0.34482002|144.94571|+0.15759277|
|10|5000|63.340282|+8.236473|179.4829|-2.2660217|
|10|10000|106.96179|-0.34316254|228.28786|+6.026947|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.4655323|-0.17244744|127.728584|-1.5105057|
|1|5000|5.2069798|-0.051792145|137.72319|-3.0079498|
|1|10000|10.603535|+0.31034088|142.71942|-3.1016998|
|1|50000|51.19006|-5.4338074|179.14333|+2.4319916|
|1|100000|100.378784|-0.6309433|231.37285|-8.3671875|
|10|100|2.2931898|+0.03450322|134.57597|+1.131607|
|10|500|5.965482|-0.06913376|138.50499|-0.6733551|
|10|1000|11.620857|-0.43089676|146.6813|+1.8931885|
|10|5000|57.657017|+2.5532074|180.23033|-1.5185852|
|10|10000|110.68693|+3.3819733|228.78677|+6.5258636|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.5173063|-0.12067342|132.01967|+2.7805786|
|1|5000|5.20705|-0.05172205|138.07031|-2.6608276|
|1|10000|10.172568|-0.120625496|142.39629|-3.4248352|
|1|50000|54.81035|-1.8135185|180.39804|+3.6867065|
|1|100000|101.98105|+0.9713211|241.82852|+2.0884857|
|10|100|2.2241652|-0.03452134|131.36975|-2.0746155|
|10|500|6.068987|+0.0343709|137.5594|-1.6189423|
|10|1000|11.775646|-0.2761078|147.18279|+2.3946686|
|10|5000|55.00051|-0.10329819|181.45474|-0.2941742|
|10|10000|117.20588|+9.900925|219.50108|-2.7598267|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.4310515|-0.034543037|128.69221|+0.572876|
|1|5000|5.3102336|-0.00025463104|138.5955|-2.53125|
|1|10000|9.552|-1.1032248|142.69962|-4.4647217|
|1|50000|52.32131|+2.6152916|178.78569|+3.5832367|
|1|100000|96.33502|-6.146843|234.60655|-15.124054|
|10|100|2.051837|+0.103438735|132.1087|-0.85614014|
|10|500|5.9310784|-0.20688486|138.11691|-2.3952026|
|10|1000|11.810618|+0.06900978|149.77936|+1.0747528|
|10|5000|57.104645|+1.2604294|180.15773|+2.947998|
|10|10000|102.61859|-3.0547104|221.79755|-4.1924896|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.4827787|+0.017184138|130.68575|+2.5664062|
|1|5000|5.155184|-0.15530443|137.97076|-3.1559906|
|1|10000|10.051814|-0.6034107|145.9534|-1.2109375|
|1|50000|48.432014|-1.2740021|175.44962|+0.24716187|
|1|100000|97.1885|-5.2933655|232.1545|-17.57611|
|10|100|2.0863307|+0.13793242|133.58109|+0.61624146|
|10|500|5.7414503|-0.396513|138.96309|-1.5490265|
|10|1000|11.845062|+0.103453636|148.80482|+0.10021973|
|10|5000|68.00381|+12.159592|180.6347|+3.4249725|
|10|10000|104.19052|-1.4827805|222.04202|-3.9480133|

**Comment by djaglowski** [comment]
| Log Files | Logs / Second | CPU Avg (%) | CPU Avg Δ (%) | Memory Avg (MB) | Memory Avg Δ (MB) |
| --- | --- | --- | --- | --- | --- |
|1|1000|1.3965431|-0.10341632|127.31331|-0.7265625|
|1|5000|5.1724586|+0.051802635|136.46794|-1.2897339|
|1|10000|10.327667|+0.15496635|144.99232|+0.953537|
|1|50000|53.51831|+3.7063065|175.35776|+0.21350098|
|1|100000|105.31265|+8.533966|230.6211|-4.3619385|
|10|100|1.9310135|-0.13792765|134.55617|+1.4388428|
|10|500|5.9138374|-0.22411585|138.28435|-1.6119385|
|10|1000|12.034722|-0.103271484|147.9274|+3.8060303|
|10|5000|54.81057|-8.31308|176.58945|-11.015488|
|10|10000|108.38498|+2.09964|216.9387|-10.659897|

File: cmd/stanza/version.go [hunk 1]
```diff
@@ -12,7 +15,7 @@ func NewVersionCommand() *cobra.Command {
 		Args:  cobra.NoArgs,
 		Short: "Print the stanza version",
 		Run: func(_ *cobra.Command, _ []string) {
-			println(version.GetVersion())
+			fmt.Println(version.GetVersion(), runtime.GOOS, runtime.GOARCH)
```
Comment (schmikei) [inline cmd/stanza/version.go:18]: Nice information to include

**Review by schmikei — APPROVED** [review]
This looks good to me.

**Pull request merged**